src/main/java/htsjdk/samtools/GATKReadSparkCodec.java

```diff
@@ -0,0 +1,196 @@
+/*
+ * The MIT License
+ *
+ * Copyright (c) 2009 The Broad Institute
+ *
```

Comment:
We should remove this license if it's not legally required.  I'm assuming it's not since broad is the original copyright owner.